src/submission/schema/schema-manager.ts

```diff
@@ -222,22 +235,25 @@ namespace MigrationManager {
       throw new Error('failed to submit migration');
     }
 
-    if (dryRun) {
-      const result = await startMigration(savedMigration);
-      return result;
-    }
+    return await runMigrationSyncOrAsync(sync || dryRun || false, savedMigration);
```

Comment:
if `sync || dryRun` are false no need to OR with `false`

Reply:
right typescript complained that both `sync` & `dryRun` could be undefined, gonna use `savedMigration.dryRun`